server/service/src/sync/translations/store_preference.rs

```diff
@@ -50,6 +54,12 @@ impl SyncTranslation for StorePreferenceTranslation {
             id: data.id,
             r#type,
             pack_to_one: data.data.pack_to_one,
+            requisitions_require_supplier_authorisation: data
```

Comment:
nit: I kind of like destructure and spread operator for these, when it was just one I didn't make a comment, but I think would be nice here now

Reply:
much nicer - thanks

Reply:
Though I'm disappointed that the strict typing means that I can't do this: 
```
        let result = StorePreferenceRow { id, r#type, ..data };
```